### DISCUSSION TITLE
Source Admin Credentials Directly

### DISCUSSION THREAD

**Discussion by dargmuesli** [strapi/strapi#6518] [Legacy Topics]
- [x] **I have created my request on the Product Board before I submitted this issue**
- [x] **I have looked at all the other requests on the Product Board before I submitted this issue**

Allow admin credentials to be loaded from file (Docker secrets) so that there is no dashboard form shown, which could be used by an attacker if they are quicker to register as an admin than the real admin themself.

**Comment by derrickmehaffy** [comment]
I find this attack vector extremely unlikely, and you wouldn't be able to register another admin. It would more likely present as an error to the user.

On that note you could call the register controller inside the bootstrap if you wanted to pre-populate an admin user.

For now I'll move this to a discussion.
